src/Web/MASA.PM.Web.Admin/Pages/Home/Overview.razor

```diff
@@ -56,7 +56,7 @@
                     @foreach (var cluster in _clusters)
                     {
                         StringNumber envClusterId = (StringNumber)cluster.EnvironmentClusterId;
-                        <MTab Style="height:40px; max-width:150px;" Value="envClusterId">
+                        <MTab Value="envClusterId">
```

Comment:
If the width is not given, the width will be adaptive and does not conform to the design